docs/CompatibleSoftware.rst

```diff
@@ -19,63 +19,191 @@ application, please contact ocio-dev as well.
 After Effects
 *************
```

Comment:
Let's add this one line description back in to clarify that this is not a native integration:

```
An OpenColorIO plugin is available for use in After Effects.
```